vfo/vfo.py

```diff
@@ -143,9 +143,9 @@ def createODB(*argv, Nmodes=0, deltaT=0.0, monitorEleTags=[], monitorEleType="no
 # 		EleIntPointsFile = os.path.join(LoadCaseDir,"EleIntPoints_All.out")	
 
 		# Save recorders in the ODB folder
-		ops.recorder('Node', '-file', NodeDispFile,  '-time', '-dT', deltaT, '-node', *nodeList, '-dof',*dofList_node, 'disp')
-		ops.recorder('Node', '-file', ReactionFile,  '-time', '-dT', deltaT, '-node', *nodeList, '-dof',*dofList_node, 'reaction')
-		
+		ops.recorder('Node', '-file', NodeDispFile,  '-time', '-dT', deltaT, '-closeOnWrite', '-node', *nodeList, '-dof',*dofList_node, 'disp')
```

Comment:
Adding -'closeOnWrite' will significantly slow down the analysis performance of large models. We will stick to 'wipe()' after the analysis in order to plot the results.